spec/factories/songs.rb

```diff
@@ -3,5 +3,6 @@
     name 'Dirty Diana'
     artist
     spotify_uri { SecureRandom.hex(10) }
+    plays 3001
```

Comment:
The you can have testcases for popular songs, unpopular songs, almost popular ones, songs without and plays, etc really easily.